Sources/SourceControl/GitRepository.swift

```diff
@@ -914,6 +914,12 @@ public final class GitRepository: Repository, WorkingCheckout {
             }
         }
     }
+
+    /// Read a symbolic link.
+    func readLink(hash: Hash) throws -> String {
+        return try callGit("cat-file", "-p", String(describing: hash.bytes),
+                           failureMessage: "Couldn't read '\(String(describing: hash.bytes))'")
```

Comment:
Formatting nit: this is what we have codified in the `.swiftformat` config.

```suggestion
        return try callGit(
            "cat-file", "-p", String(describing: hash.bytes),
            failureMessage: "Couldn't read '\(String(describing: hash.bytes))'"
        )
```